ArduCopter/mode_loiter.cpp

```diff
@@ -125,17 +125,21 @@ void ModeLoiter::run()
 
     case AltHoldModeState::MotorStopped:
         attitude_control->reset_rate_controller_I_terms();
-        attitude_control->reset_yaw_target_and_rate();
+        attitude_control->reset_yaw_target_and_rate(true);
```

Comment:
this is not a change in behaviour, default value is true.

Reply:
Ok.  I wasn't sure that I needed to do this.  I saw that it was done in one place so I thought I needed to do it in all places. I can remove it.

Reply:
fixed. Thanks.